dristi-monolith/domain-integration/src/main/resources/treasury/db/migration/main/V20240708134500__treasury_backend_ddl.sql

```
@@ -0,0 +1,60 @@
+    ADD COLUMN reSubmissionBreakDown JSONB;
+ALTER TABLE treasury_head_breakup_data
+    ADD COLUMN lastModifiedTime BIGINT;
+ALTER TABLE treasury_head_breakup_data
+    ADD COLUMN lastSubmissionConsumerCode VARCHAR(64) NULL;
+
+    ALTER TABLE treasury_head_breakup_data
+    RENAME COLUMN reSubmissionBreakDown TO finalCalcPostResubmission;
+
+
+
+ALTER TABLE auth_sek_session_data ADD COLUMN request_blob jsonb NULL;
```

SQLFluff (4.1.0)

[error] Too many consecutive blank lines.

(LT15)